code/script.js

```diff
@@ -1,9 +1,20 @@
+var sectionOne=document.getElementById("section1");
+var sectionTwo=document.getElementById("section2");
+var sectionThree=document.getElementById("section3");
```

Comment:
You can use `const` instead of `var` here! When declaring a variable you want to use 1) const, 2) let and third 3) var. In some cases var is needed!